lib/helpers.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Helper functions
+
+# Extract Object or objects matching .name, including deduplication
+# $1: item name, prepended or not with deduplication mark
+array_from_name() {
+  item_name=$(echo $1 | sed "s/$DEDUP_MARK //")
+  echo "$ITEMS" | jq -r ". | map(select(.name == \"$item_name\"))"
+}
+
+# Evaluate if the arrat passed has a single element. Call show_full_items() if it's not
+# $1: Array of items
+not_unique() {
+  item_count=$(echo "$1" | jq -r '. | length')
+  ! [[ $item_count -eq 1 ]]
+}
+
+# Pipe a document and deduplicate lines.
+# Mark those duplicated by prepending $DEDUP_MARK
+dedup_lines() {
+  sort | uniq -c \
+  | sed 's/^[[:space:]]*//' \
+  | sed "s/\\([2-9]+*\\) /$DEDUP_MARK /" \
```

Comment:
this way all the doubles containing a value between 10 and 20 and every multiplication of 10 won't be substituted.

My suggestion:
```bash
sort | uniq -c \
| sed "s/^\s*1 //" \
| sed -r "s/^\s*[0-9]+ /$DEDUP_MARK /"

Reply:
Fair point. Although I don't think anyone will handle 10*k passwords with the same name, I'll be sure this is fixed. Thanks for the suggestion!